dependencies/src/main/java/com/google/cloud/tools/opensource/classpath/StaticLinkageCheckReport.java

```diff
@@ -17,16 +17,29 @@
 package com.google.cloud.tools.opensource.classpath;
 
 import com.google.auto.value.AutoValue;
+import com.google.common.annotations.VisibleForTesting;
 import com.google.common.collect.ImmutableList;
 
 /**
  * The result of a static linkage check.
  */
 @AutoValue
-abstract class StaticLinkageCheckReport {
+public abstract class StaticLinkageCheckReport {
+
   abstract ImmutableList<JarLinkageReport> getJarLinkageReports();
 
-  static StaticLinkageCheckReport create(Iterable<JarLinkageReport> jarLinkageReports) {
+  @VisibleForTesting
+  public static StaticLinkageCheckReport create(Iterable<JarLinkageReport> jarLinkageReports) {
     return new AutoValue_StaticLinkageCheckReport(ImmutableList.copyOf(jarLinkageReports));
   }
+  
+  @Override
+  public String toString() {
+    StringBuilder builder = new StringBuilder();
+    for (JarLinkageReport jarLinkageReport : getJarLinkageReports()) {
```

Comment:
How do you feel about

`return getJarLinkageReports().stream().map(report -> report.toString() + "\n").collect(joining());`